chore: remove bundles from monorepro

DerCalli opened the pull request:
**Changelog/Description**

The following bundles have been removed from the monorepo:
- customer-token-manager
- customer-token-manager-one-time-password-connector
- one-time-password
- one-time-password-email-connector
- one-time-password-extension
- one-time-password-rest-api

julianzimmermann commented on bundles/jellyfish-sales-order/src/FondOfOryx/Zed/JellyfishSalesOrder/Business/Model/Mapper/JellyfishOrderAddressMapper.php:
```diff
@@ -31,6 +31,8 @@ public function fromSalesOrderAddress(SpySalesOrderAddress $salesOrderAddress):
             ->fromArray($salesOrderAddress->toArray(), true);
 
         $jellyfishOrderAddressTransfer->setId($salesOrderAddress->getIdSalesOrderAddress())
+            ->setFirstName(null)
```
Würde der das nicht auch so für B2C überschreiben?

Commit:
remove:
- customer-token-manager
- customer-token-manager-one-time-password-connector
- one-time-password
- one-time-password-email-connector
- one-time-password-extension
- one-time-password-rest-api

Commit:
Add removed bundles to require part of composer.json

sourcecube reviewed (approved):
LGTM
